parm/config/gefs/config.extractvars

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env bash                                                                                                                                                                                                                     
+
+########## config.extractvars ##########
+# Extractvars specific
+
+echo "BEGIN: config.extractvars"
+
+. "${EXPDIR}/config.resources" extractvars
+
+export compress_ocn=0 #1: compress extracted ocean product, 0: do not compress extracted ocean product
+export compress_ice=0 #1: compress extracted ice product, 0: do not compress extracted ice product
+export FHMAX=120
+
+export FHMAXHF=60 #The forecast length of the high output frequency (in hours) for atmos
```

Comment:
Will add an error check for this just in case a user sets FHMAX<FHMAXHF.

Reply:
To reduce the number of new variables for this task, I will use `FHMAX_HF_GFS` from config.base instead of `FHMAXHF`. These two variables serve the same purpose.